docs/book/core-concepts.md

```diff
@@ -172,9 +170,9 @@ On a high level, when data is read from an **artifact** the results are persiste
 
 A few rules apply:
 
-* Every **orchestrator** (local, Google Cloud VMs, etc) can run all **pipeline steps**, including training.
-* **Orchestrators** have a selection of compatible **processing backends**.
-* **Pipelines** can be configured to utilize more powerful **processing** (e.g. distributed) and **training** (e.g. Google AI Platform) **executors**.
+- Every **orchestrator** (local, Google Cloud VMs, etc) can run all **pipeline steps**, including training.
```

Comment:
Are these all automatic from your linter? We should standardize it otherwise it becomes hard to review markdown. Can you share the tools your using? Maybe we can add a pre-commit hook?

Reply:
Yeah sorry about that.

I think it is `prettier` which is taking out extra spaces on the ends of lines.

https://prettier.io/docs/en/precommit.html shows how it'd integrate with `precommit`.

I imagine there will be some overlap with some things we already have? For markdown, my settings are:

```
  "arrowParens": "always",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "es5",
  "useTabs": false,
  "vueIndentScriptAndStyle": false,
  "filepath": "/Users/strickvl/coding/zenml/repos/zenml/docs/book/guides/low-level-api/chapter-7.md",
  "parser": "markdown"
```

Reply:
Would you mind adding it to the whole dev cycle? Including adding a new `script` and editing our pre-commit hook?

Reply:
Added in a separate branch.